### ISSUE TITLE
Create facet: Media format = edm:type

### ISSUE THREAD

**Issue by marlo-longley** [sul-dlss/vt-arclight#143]
Ref #29 

Question: where in the EAD is this data?
Is this the right data? `<physdesc id="aspace_4550e0bf19381a53ac6c8bf46200190c">TEXT</physdesc>`
Can we select for physdesc's that have ID's and this is correct?
Right now it looks like all the data is TEXT -- if so do we want to facet on this, and is this correct?

Note that the edm:type values in the current spreadsheet need to be cleaned up before we can use them:

At least a few records have the value 3D, which doesn't seem correct
All values are currently in uppercase; we'll want to display them with init caps
See https://github.com/sul-dlss/nta-arclight/issues/10 for more about this one. I think @laurensorensen might be looking into this.

Example/discovery: Defense for Hess Part 2 has `<physdesc id="aspace_5e3b7bf545c88e0535488da73f5fcb89">Sound</physdesc>` indicatind sound type but why does Part 1 not have that?
```
<c id="aspace_59e25269b4f9cc85fe15e19664484df2" level="item">
              <did>
                <unittitle>Defense for Hess. Part 1</unittitle>
                <unitid>H-5260_1732A_1</unitid>
                <physdesc altrender="whole">
                  <extent altrender="materialtype spaceoccupied">1 item(s)</extent>
                  <physfacet>1899-12-31T00:16:28+00:00</physfacet>
                </physdesc>
</c>
<c id="aspace_204269ab5fa5c3f771e0ceef3849bf78" level="item">
              <did>
                <unittitle>Defense for Hess. Part 2</unittitle>
                <unitid>H-5260_1733A_1</unitid>
                <physdesc altrender="whole">
                  <extent altrender="materialtype spaceoccupied">1 item(s)</extent>
                  <physfacet>1899-12-31T00:15:58+00:00</physfacet>
                </physdesc>
                <physdesc id="aspace_5e3b7bf545c88e0535488da73f5fcb89">Sound</physdesc>              
</c>

```

**Comment by laurensorensen** [comment]
I am pretty sure for 3D tagged records they are excluded from the digitization work, so I've removed in most of the data (I'll keep an eye out as I clean up the data, though). 

Not sure why Part 1 Document for Hess doesn't have Sound tag... It is in the spreadsheet so may have been an error on my part with copy/paste. 

I will be doing the clean up on edm:type as a I prep the 12 series sheets for uploader tool. 
I'm changing =
TEXT to Text
VIDEO to Moving Image
IMAGES to Images
SOUND to Sound

The field should appear in Sample data now, as Physical Description (so I am pretty certain the field you indicated above is correct @marlo-longley, `<physdesc id="aspace_4550e0bf19381a53ac6c8bf46200190c">TEXT</physdesc>`) 

I put the mappings in the EAD/DACS tab here (have the field name from ICJ spreadsheet and column mapped to field in ASpace): https://docs.google.com/spreadsheets/d/1Xz0hq28tyhESPTAcpowhlhw8JHNUManDWjP3hXZDkPg/edit#gid=185071693

**Comment by marlo-longley** [comment]
Closing in favor of clearer tickets.
